kore/src/Kore/Step/Simplification/Pattern.hs

```diff
@@ -28,13 +34,14 @@ import Kore.Step.Simplification.Simplify
     , simplifyConditionalTermToOr
     )
 
-simplifyAndRemoveTopExists
-    :: SimplifierVariable variable
+simplifyTopConfigurationAndRemoveTopExists
```

Comment:
Maybe we could call this only `simplifyTopConfiguration`? Right now, there are two special things we do when simplifying the top configuration:

1. Use `Condition.topTODO` as the top-level condition.
1. Remove any existential quantifiers at the top.

I imagine we might do more special things to the top-level configuration in the future, without changing the name of this function. A comment would be good, listing the special items. Anyway, if you disagree then feel free to disregard this comment.

Reply:
Done.